src/plugins/vis_type_vega/public/data_model/utils.test.js

```diff
@@ -91,4 +87,30 @@ describe('Utils.handleInvalidQuery', () => {
       }).toThrowError();
     });
   });
+
+  test('should identify object contains function properties', async () => {
```

Comment:
Good catch! For test completeness, could we have a test case whose result is `false`?

Reply:
Added additional tests to cover negative cases.